docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jupyter/datascience-notebook:latest
+FROM jupyter/datascience-notebook:fa77fe99579b
```

Comment:
Is there a specific reason to use a fixed tag instead of a version, or simply the latest?